src/client/scripts/game/chess/movepiece.js

```diff
@@ -248,11 +248,22 @@ const movepiece = (function(){
      * - `doGameOverChecks`: Whether game-over checks such as checkmate, or other win conditions, are performed for this move.
      */
     function flipWhosTurn(gamefile, { pushClock = true, doGameOverChecks = true } = {}) {
-        gamefile.whosTurn = math.getOppositeColor(gamefile.whosTurn);
+        gamefile.whosTurn = getWhosTurn(gamefile);
```

Comment:
I know this is just a draft, but this function name is immediately confusing me. What if we called it something like `getWhosNextTurn()` instead?

Reply:
This has now been merged with the gamefile utility function `getWhosTurnAtMoveIndex`